spec/indexers/sufia/collection_indexer_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe Sufia::CollectionIndexer do
+  describe "fontawesome default icon" do
```

Comment:
The default collection icon is/was a font awesome icon. Perhaps a better describe would be "default collection icon?"